src/styles/index.css

```diff
@@ -0,0 +1,125 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+}
+
+.card {
+  width: 200px;
+  height: 406px;
+  padding: 16px;
+  border-radius: 5px;
+  color: #060b35;
+  box-sizing: border-box;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+}
+
+.discribe {
```

Comment:
The class name 'discribe' appears to be a typo. Consider changing it to 'describe' to avoid confusion and maintain consistency.